sdk/js/index.ts

```diff
@@ -205,4 +205,24 @@ export default class StreamManager {
       onLogs,
     });
   }
+
+  async cancel(
+    streamId: number,
+    reason: string,
+    creator: Address
+  ): Promise<bigint> {
+    if (!this.walletClient)
+      throw new Error("Error cancelling stream: wallet client is not set");
+    const result = await this.walletClient.writeContract({
+      chain: null,
+      address: this.address,
+      abi: StreamManagerContractType.abi as Abi,
+      functionName: "cancel_stream",
+      args: [streamId, reason, creator],
+      account: creator,
+    });
+    if (result === undefined)
+      throw new Error("Error while processing transactions");
+    return BigInt(result);
```

Comment:
issue: I believe `writeContract()` returns a tx hash here, not an int.  Unless we're using BigInt instead of strings for hashes, this probably isn't what we want.  The call for `create()` is an example:

https://github.com/ApeWorX/ApePay/blob/8af998c05ca5398ab8fd3c66419cc499171aa730/sdk/js/index.ts#L157-L167